examples/cfd/example_diffusion.py

```diff
@@ -1,7 +1,16 @@
 """
```

Comment:
Because it's pretty much superseded by tutorial 03 (it's the same equation, but with a different test). I'd like to keep it around because it has our only `lambdify` example, but I've added a note at the top that it this is a legacy example.